samples/voting/README.md

```diff
@@ -9,11 +9,44 @@ The project should be immediately runnable by calling `tye run` from the directo
 
 A few things need to be configured before deploying to Kubernetes.
 
-- Setting up Redis. A connection string needs to be provided to connect to Redis. You can follow our tutorial on [setting up redis in your cluster](../../docs/tutorials/hello-tye/02_add_redis.md).
-- Setting up postgresql. A connection string eventually needs to be provided to tye for postgresql.
-- Deploying the ingress.yaml by calling:
+- [Create an Azure container registry using the Azure portal](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal) or you can use your public [docker hub account](https://hub.docker.com/)
+
+- `Redis` can be deployed using the below command :
 
     ```
-    kubectl apply -f ingress.yml
+    kubectl apply -f https://raw.githubusercontent.com/dotnet/tye/master/docs/tutorials/hello-tye/redis.yaml
+    ```
+- `Postgresql` can be installed by the following command :
+
+    ```
+    kubectl apply -f https://raw.githubusercontent.com/dotnet/tye/master/docs/tutorials/hello-tye/postgres.yaml
     ```
+
+- Once the deployment is done, you need to keep a note of the below connection strings :
+
+    -  `redis:6379`
+    -  `Server=postgres;Port=5432;User Id=postgres;Password=pass@word1;`
+
+>! NOTE: If you want, you can modify the password in `postgres` yaml.
+
+- You need to install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/) in your Kubernetes cluster. 
```

Comment:
Actually, we should be able to remove this step here, as tye now supports deploying ingresses, including installing the NGINX ingress controller. Can you do the following:
- Remove the ingress.yaml and translate it to the equivalent tye.yaml ingress definition. See https://github.com/dotnet/tye/blob/master/samples/apps-with-ingress/tye.yaml as an example.
- Remove these bullets about deploying the ingress.

Reply:
@jkotalik - Yes, correct. Have done the following changes : 

- Removed `ingress.yaml` and updated  `tye` yaml  to deploy ingress. 
- Removed the ingress deploy section.